docker-compose/docker-compose-v2.1.yaml

```diff
@@ -2,7 +2,7 @@ version: '2.1'
 
 services:
   unifiedpushserver:
-    image: docker.io/aerogear/unifiedpush-wildfly:2.1.0
+    image: aerogear/unifiedpush-wildfly-plain:2.2.2-SNAPSHOT
```

Comment:
Not part of this PR, but I think it would be good to start publishing this to https://quay.io/organization/aerogear instead

Reply:
Good catch, we can change it before merge, anyway it's using temp SNAPSHOT and will be replaced by the released one!